### ISSUE TITLE
Move templates/examples out of this repo

### ISSUE THREAD

**Issue by potatodepaulo** [nearprotocol/NEARStudio#107]
The templates that we run in Studio should be in their own repository.
The best way to get this done is probably to have some service that moves files around as a part of our build process. 

One solution: https://github.com/livechat/copycat

There are probably simpler solutions than spinning up a node server.

**Comment by vgrichina** [comment]
Yeah, the simplest solution is to not move templates out of studio repo.
